ts/sagas/wallet/pagopaApis.ts

```diff
@@ -88,6 +89,7 @@ export function* getWalletsV2(
     }
   } catch (error) {
     yield put(fetchWalletsFailure(error));
+    yield put(sessionExpired());
```

Comment:
In this way we consider a session invalid when an exception is raised.

Instead we have to consider a session invalid **only** when the response is right and the http status code is [401](https://httpstatuses.com/401) 
see https://github.com/pagopa/io-app/blob/8fe7cff334b89bf919fd9dd45843c37d78079ee3/ts/sagas/profile.ts#L65